src/Arcus.Messaging.Tests.Workers.ServiceBus.Topic/Arcus.Messaging.Tests.Workers.ServiceBus.Topic.csproj

```diff
@@ -8,14 +8,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Arcus.EventGrid.Publishing" Version="3.0.0" />
-    <PackageReference Include="Arcus.EventGrid.Testing" Version="3.0.0" />
-    <PackageReference Include="Guard.NET" Version="1.2.0" />
-    <PackageReference Include="Microsoft.Extensions.Configuration" Version="3.1.0" />
-    <PackageReference Include="Microsoft.Extensions.Hosting" Version="3.1.0" />
+    <PackageReference Include="Arcus.EventGrid.Publishing" Version="[3.1.0,4.0.0)" />
+    <PackageReference Include="Arcus.EventGrid.Testing" Version="[3.1.0,4.0.0)" />
+    <PackageReference Include="Guard.NET" Version="[1.2.0,2.0.0)" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="[3.1.8,5.0.0)" />
+    <PackageReference Include="Microsoft.Extensions.Hosting" Version="[3.1.8,5.0.0)" />
     <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="4.4.2" />
     <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.9.5" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="Newtonsoft.Json" Version="[12.0.0,13.0.0)" />
```

Comment:
Not sure if this flexible package ranges for test projects is useful.

Reply:
I think that the Polly NuGet package build failure is also because these ranges include possible invalid combinations. Polly is not even explicitly part of this library, it's only used indirectly via possible our Arcus.Security package or other dependencies. A reference to Polly directly could possible help with this.

Reply:
> Not sure if this flexible package ranges for test projects is useful.

I have to or it's not in line with our other projects. I could remove them an manually keep them aligned though.



> I think that the Polly NuGet package build failure is also because these ranges include possible invalid combinations. Polly is not even explicitly part of this library, it's only used indirectly via possible our Arcus.Security package or other dependencies. A reference to Polly directly could possible help with this.

Good idea!

Reply:
Also good to mention, I succeeded in reproducing this locally. In case you're wondering and don't want to wait on the build.